docs/standard/choosing-core-framework-server.md

```diff
@@ -1,108 +1,106 @@
+You can then host your Docker containers in your own Linux or Windows infrastructure, or use a cloud service such as [Azure Container Service](https://azure.microsoft.com/services/container-service/), which can manage, orchestrate, and scale your container-based application in the cloud.
 
-### A need for high performance and scalable systems
+### A need for high-performance and scalable systems
 
-When your system needs the best possible performance and scalability, .NET Core and ASP.NET Core are your best options. ASP.NET Core outperforms ASP.NET by a factor of 10, and it leads other popular industry technologies for microservices such as Java servlets, Go and node.js.
+When your system needs the best possible performance and scalability, .NET Core and ASP.NET Core are your best options. ASP.NET Core outperforms ASP.NET by a factor of 10, and it leads other popular industry technologies for microservices such as Java servlets, Go, and Node.js.
 
-This is especially relevant for microservices architectures, where you could have hundreds of microservices running. With ASP.NET Core you’d be able to run your system with a much lower number of servers/VMs, ultimately saving costs in infrastructure and hosting.
+This is especially relevant for microservices architectures, where you could have hundreds of microservices running. With ASP.NET Core, you’d be able to run your system with a much lower number of servers/VMs, ultimately saving costs in infrastructure and hosting.
 
 ### A need for side by side of .NET versions per application level
 
-If you want to be able to install applications with dependencies on different versions of frameworks in .NET, you need to use .NET Core, which provides 100% side-by-side. Easy side-by-side installation of different versions of .NET Core on the same machine allows you to have multiple services on the same server, each of them on its own version of .NET Core, eliminating risks and saving money in application upgrades and IT operations.
+If you want to be able to install applications with dependencies on different versions of .NET, we recommend you use .NET Core. .NET Core offers easy side-by-side installation of different versions of the .NET Core runtime on the same machine, allowing you to have multiple services on the same server, each of them on its own version of .NET Core, eliminating risks and saving money in application upgrades and IT operations.
 
 ## When to choose .NET Framework
 
-While .NET Core offers significant benefits for new applications and application patterns, the .NET Framework will continue to be the natural choice for many existing scenarios and as such, it won’t be replaced by .NET Core for all server applications.
+While .NET Core offers significant benefits for new applications and application patterns, the .NET Framework continues to be the natural choice for many existing scenarios and as such, it isn't replaced by .NET Core for all server applications.
```

Comment:
~While~ .NET Core offers significant benefits for new applications and application patterns. However, the .NET Framework continues to be the natural choice for many existing scenarios.  The .NET Framework  isn't replaced by .NET Core for all server applications.